src/Raven.Server/Documents/Indexes/MapReduce/ReduceMapResultsBase.cs

```diff
@@ -197,8 +198,8 @@ protected ReduceMapResultsBase(T indexDefinition, IndexStorage indexStorage, Met
                         {
                             var message = $"Failed to execute reduce function for reduce key '{modifiedState.Tree.Name}' on a branch page #{page} of '{_indexDefinition.Name}' index.";
 
-                            if (Log.IsWarnEnabled)
-                                Log.WarnException(message, e);
+                            if (_logger.IsOperationsEnabled)
```

Comment:
This isn't an operator level error. It is already reported in the indexes error, and should go to the regular log